CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+## [0.2.2] - 2023-02-20
```

Comment:
This should go into `## [Unreleased]`, our CI/CD automatically handles the changelog and version bumping.

Reply:
ok, changed